app/javascript/components/recipes/index/recipes-container.vue

```diff
@@ -99,13 +106,32 @@ export default {
     }
   },
   methods: {
+    updateRecipes() {
+      this.recipesFiltered = [];
+      for (let i = 0; i < this.recipesAll.length; i ++) {
+        if (this.recipesAll[i].portions >= this.filters.portions.min) {
+          if (this.recipesAll[i].portions <= this.filters.portions.max) {
+            this.recipesFiltered.push(this.recipesAll[i]);
+          }
+        }
+      }
+      this.recipesShow = this.recipesFiltered;
```

Comment:
esto lo podrías hacer con un filter y con un and, en general en javascript se intenta evitar usar fors directos a menos que sea estrictamente necesario, queda más legible usando filter, map, forEach, find, etc:
```js
this.recipesAll.filter(
  recipe => recipe.portions >= this.filters.portions.min && recipe.portions <= this.filters.portions.max,
);
```